CGRPCZlib.podspec

```diff
@@ -0,0 +1,19 @@
+Pod::Spec.new do |s|
+
+    s.name = 'CGRPCZlib'
+    s.module_name = 'CGRPCZlib'
+    s.version = '1.0.0-alpha.11'
+    s.license = { :type => 'Apache 2.0', :file => 'LICENSE' }
+    s.summary = 'Swift gRPC code generator plugin and runtime library'
+    s.homepage = 'https://www.grpc.io'
+    s.authors  = { 'The gRPC contributors' => 'grpc-packages@google.com' }
+
+    s.source = { :git => 'https://github.com/grpc/grpc-swift.git', :tag => s.version }
+
+    s.swift_version = '5.0'
+    s.ios.deployment_target = '10.0'
+    s.osx.deployment_target = '10.12'
```

Comment:
Given that NIO now supports 10.10+ (I think), should we adopt that as our deployment target here, too?

Reply:
Definitely! Good catch!